examples/hostinstall/centos7/openc3_install_openc3.sh

```diff
@@ -52,7 +52,7 @@ GEMS="$SCRIPT_DIR/../../../openc3-cosmos-init/plugins/gems/"
 OPENC3_RELEASE_VERSION=5.21.0-beta0
 
 mkdir -p ${GEMS}
-cd ${PLUGINS}openc3-tool-base && yarn install && yarn run build && rake build VERSION=${OPENC3_RELEASE_VERSION} && mv *.gem ${GEMS}
+cd ${PLUGINS}packages/openc3-tool-base && yarn run build && rake build VERSION=${OPENC3_RELEASE_VERSION} && mv *.gem ${GEMS}
```

Comment:
Are we planning on deleting this hostinstall example?

Reply:
I thought you already did that?
If not, then yes I want that removed for the 6.0 release.